03LessonThree/in-class-project/css/daily-planet-style.css

```diff
@@ -7,18 +7,55 @@ header, nav, main, section, aside, footer {
     padding: 5px;
     margin-top: 10px;
 }
-
 section, aside {
     float: left;
 }
-
+h1{
+  font-size:300%;
+  display:inline;
+  margin-top:-50px;
+}
 section {
-    width: 70%
+    width: 65%;
+    border:solid black 2px;
 }
 aside {
-    width: 20%;
+    width: 30%;
+    border:solid black 2px;
 }
 
 footer {
-    clear: both;
-}
\ No newline at end of file
+    clear:both;
+    background-color:#a0a0a0;
+    width:100%;
+    text-align:center;
+    border:solid black 2px;
+}
+footer p{
+  display:inline;
+  padding:25px;
+}
+nav{
+  float:right;
+  font-size:130%;
+  padding:25px;
+}
+header{
+  background-color:#a0a0a0;
+  border:2px solid black;
+}
+figure p{
+  text-align:center;
+}
+footer a{
+  padding:10px;
+}
+figcaption{
+  text-align:center;
+  padding-left:30px;
+  text-decoration:underline;
+  font-size:120%;
+}
+aside p{
+  margin-left:30px;
+}

```

Comment:
Great job taking the CSS beyond the provided code! Keep pushing yourself with new ideas and you will only get better.